server/plugin/client.go

```diff
@@ -497,6 +497,10 @@ func (c *client) makeHTTPRequest(req *http.Request, contentType string, out inte
 	}
 	defer resp.Body.Close()
 
+	// Limit reading the response bodies to 1 MB or 1000 KB
+	// This is ideal for the responses returned by Azure DevOps APIs used here
+	resp.Body = http.MaxBytesReader(nil, resp.Body, constants.MaxBytesSizeForReadingResponseBody)
```

Comment:
It seems off to reassign `resp.Body` here. Thoughts on making this its own var?

Reply:
`resp.Body` was not used further after this so reassigned to it, but yes having a separate variable for it will make it more clear